common/web/gesture-recognizer/src/inputEventEngine.ts

```diff
@@ -1,14 +1,20 @@
-/// <reference path="inputEventCoordinate.ts" />
+/// <reference path="inputSequence.ts" />
 /// <reference path="gestureRecognizerConfiguration.ts" />
 /// <reference path="includes/events.ts" />
+/// <reference path="incomplete.ts" />
 
 namespace com.keyman.osk {
-  export type InputHandler = (coord: InputEventCoordinate) => void;
-
+  /**
+   * Supported events:
+   *
+   * - 'inputstart' - a new mouse or touch input sequence has begun.
+   *   - further events are supported on the sequence itself.
+   */
   export abstract class InputEventEngine extends EventEmitter {
-    public static readonly INPUT_UPDATE_EVENT_NAME = "inputUpdate";
+    public static readonly INPUT_START_EVENT_NAME = "inputstart";
 
     protected readonly config: Nonoptional<GestureRecognizerConfiguration>;
+    private _activeSequenceWrappers: Incomplete<InputSequence, InputSample>[] = [];
```

Comment:
What is a 'sequence wrapper'? The term `wrapper` here feels a little unnecessary. It could just be `_activeSequences`.

Reply:
This ties in with the `Incomplete` question, actually.

The design I landed on for this PR, and the reasoning, is as follows.

1) We want the ability to serialize the `InputSequence` object.  This gives us the ability to easily record input sequences for use in unit testing, so making sure the important parts transfer in and out of JSON relatively simply is fairly important.

2) For an object to be serialized, should that object itself directly include event-management overhead?  i.e, extend `EventEmitter`, which likely introduces its own internal properties that will need to be filtered, etc.

3) There will come a point when the sequence is "finalized" and will no longer mutate.  Before that... well, it's "incomplete."  The typing was hoping to make that somewhat clear via nomenclature.

------

That said... I'm currently in an implementation-design mode for the next phase of the gesture work, and I'm now not 100% set on this design as it's seen here; I'm still working through exactly what the module-consumer's "API" for receiving gesture-related updates, etc might be.

This PR's main goal is to promote the ability to track the individual touchpoints clearly and reliably, encapsulating updates for each one, individually, over time into a common lookup location.  Not so much to establish a permanent API, so I'm not _set_ on use of this wrapper design.